setup/install_gatling/30_classification/90_clamps.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+# Clamps is a module based stress testing tool. It will generate a number of
+# users on a system to simulate agents. It will then randomize their puppet
+# agent -t times to simulate a large number of agents.
+def add_clamps_group_ca(classifier, pe_infra)
+  # The Clamps::master class manages auth.conf for the agents - by setting it
+  # to allow all.  This is insecure and worth looking at overriding at some
+  # point, but this should only be used for metrics / load testing on a private
+  # network, so shouldn't be an issue short term.
+  clamps_ca_group = {
+    'name'    => 'Clamps CA',
+    'rule'    => ['or', ['=', 'name', master.node_name]], # pinned node
```

Comment:
why the "or".  There is only one rule

Reply:
This is the way the rule was defined in the original clamps classification file. I don't think it matters which logical operator is used.